mobile_verifier/src/speedtests_average.rs

```diff
@@ -180,6 +186,8 @@ impl SpeedtestTier {
 
     fn from_latency(latency: u32) -> Self {
```

Comment:
[HIP98](https://github.com/helium/HIP/blob/afd363bd261ae13d4f732dba7b62f09f2d119516/0098-mobile-subdao-quality-of-service-requirements.md) states:

| Speed Test Tier | Speedtest Multiplier | Requirement (speeds in Mbps, latency in ms)    |
| --------------- | -------------------- | ---------------------------------------------- |
| Good            | 1.00X                | 100+ Download, AND 10+ Upload, AND <50 Latency |
| Acceptable      | 0.75X                | 75+ Download, AND 8+ Upload, AND <60 Latency   |
| Degraded        | 0.50X                | 50+ Download, AND 5+ Upload, AND <75 Latency   |
| Poor            | 0.25X                | 30+ Download, AND 2+ Upload, AND <100 Latency  |
| Fail            | 0.00X                | <30 Download, OR <2 Upload, OR >100 Latency    |

Therefore the `<=`'s in the latency check should be changed to `<` to match the HIP text.

Reply:
Good eyes, fixed febdcce